cypress/integration/chi-custom-elements/drawer.spec.js

```diff
@@ -0,0 +1,113 @@
+/// <reference types="Cypress" />
+
+describe('Drawer', function() {
+
+  beforeEach(()=>{
+    cy.fixture('chidata.json').as('chidata');
+  });
```

Comment:
The beforeEach isn't really doing anything useful for this test, I think it can be removed

Reply:
+1

Reply:
Yep, I thought I already removed this but I forgot. Now it has been removed.